src/sign-verify/clu_verify.c

```diff
@@ -51,11 +54,17 @@ byte* wolfCLU_generate_public_key_rsa(char* privKey, byte* outBuf,
     /* read in and store private key */
 
     privKeyFile = fopen(privKey, "rb");
+    if (privKeyFile == NULL) {
+        printf("Unable to open file %s\n", privKey);
+        return NULL;
+    }
     fseek(privKeyFile, 0, SEEK_END);
     privFileSz = ftell(privKeyFile);
     keyBuf = malloc(privFileSz);
```

Comment:
`malloc` -> `XMALLOC`. Will stop making this comment, but this should be fixed everywhere, I think.